api/v3/Job/Iatsverify.php

```diff
@@ -183,7 +183,7 @@ function civicrm_api3_job_iatsverify($params) {
             // Restore source field and trxn_id that completetransaction overwrites
             civicrm_api3('contribution', 'create', array(
               'id' => $contribution['id'],
-              'source' => $contribution['source'],
+              'source' => ($contribution['contribution_source'] ?? $contribution['source']),
```

Comment:
The left-hand-side would also need to be contribution_source. Source doesn't seem to work.

Reply:
Actually I take that back. Nevermind.

Reply:
Yeah, the request parameters seem to be unchanged, it's only the api response that's changed. Pretty random ...